From 22064443b05288047c2edbaf8afac43cc9031427 Mon Sep 17 00:00:00 2001
From: Benda Xu <heroxbd@gentoo.org>
Date: Thu, 11 Sep 2014 18:20:44 +0900
Subject: disable class and collaboration graph generation

Forwarded: no need
Description: The graphviz-generated class and collaboration graph
 occupies 1GB disk space.  It is too big to deal with.  We turn them
 off only to save space.  Output image format is specified as svg to
 further save space (bug #751947).
---
 doxygen.cfg | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/doxygen.cfg b/doxygen.cfg
index 0fc2602..0a1cdcc 100644
--- a/doxygen.cfg
+++ b/doxygen.cfg
@@ -34,11 +34,12 @@ PREDEFINED              = DOXYGEN_SHOULD_SKIP_THIS \
                           USE_THREADS \
 			  BOOST_PP_IS_ITERATING
 ALLEXTERNALS            = YES
-CLASS_GRAPH             = YES
-COLLABORATION_GRAPH     = YES
+CLASS_GRAPH             = NO
+COLLABORATION_GRAPH     = NO
 GROUP_GRAPHS            = NO
 GRAPHICAL_HIERARCHY     = NO
 DIRECTORY_GRAPH         = NO
+DOT_IMAGE_FORMAT        = svg
 UML_LOOK                = YES
 SEARCHENGINE            = YES
 
